default/scripting/buildings/IMPERIAL_GARDENS.focs.txt

```diff
@@ -0,0 +1,28 @@
+BuildingType
+    name = "BLD_IMPERIAL_GARDENS"
+    description = "BLD_IMPERIAL_GARDENS_DESC"
+    buildcost = 150 * [[BUILDING_COST_MULTIPLIER]]
+    buildtime = 5
+    location = And [
+        Planet
+        Contains Building name = "BLD_IMPERIAL_PALACE"
+        Not Contains Building name = "BLD_IMPERIAL_GARDENS"
+        OwnedBy empire = Source.Owner
+        TargetPopulation low = 1
+    ]
+    EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
+    effectsgroups = [
+        EffectsGroup
+            scope = And [
+                Object id = Source.PlanetID
+                Planet
+                NOT Population high = 0
```

Comment:
could probably rephrase more clearly with `(LocalCandidate.Population > 0)`

Reply:
I copied that condition from AUTO_HISTORY_ANALYSER.focs.txt - should i use your proposed different style even though that makes adjacent files use inconsistent style, or do you want to change the existing script(s), too?

Reply:
Up to you. I'd prefer all consistent and clear, but tweaking existing scripts is perhaps better put in a separate pull request or commit.

Reply:
So i'll leave such tweaking for a different commit and stick to the existing style in this PR.